features/facebookImport/test/importers/personal-data-importer.test.js

```diff
@@ -89,7 +89,6 @@ test("PersonalDataImporter - importedFileName returned correctly", async () => {
         zipFile
     );
 
-    console.log(report);
```

Comment:
Ah, now it's gone - nevermind my comment on the previous PR then :D

Reply:
Removed it in the other one as well so it won't show up here after I rebased :D